synapse/cores/common.py

```diff
@@ -2347,20 +2347,6 @@ def _initPropTufo(self, tufo):
         self.addTufoProp(form, prop, **info)
         return
 
-    def _tufosByIn(self, prop, valus, limit=None):
```

Comment:
not sure i follow why this was removed in this PR ( and maybe some stuff below ) :)